ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/CloudSourceIntegrations/OcmIntegrationForm.tsx

```diff
@@ -0,0 +1,422 @@
+                            onChange={(event, value) => onChange(value, event)}
+                            onBlur={handleBlur}
+                            isDisabled={!isEditable}
+                        />
+                    </FormLabelGroup>
+                    <FormLabelGroup
+                        isRequired
+                        label="Endpoint"
+                        labelIcon={
+                            <IntegrationHelpIcon
+                                helpTitle="OpenShift Cluster Manager endpoint"
+                                helpText={
+                                    <Flex direction={{ default: 'column' }}>
+                                        <FlexItem>
+                                            The API endpoint under which OpenShift Cluster Manager
+                                            is available. Most users will not need to change the
+                                            preset value.
+                                        </FlexItem>
+                                    </Flex>
+                                }
+                                ariaLabel="Help for endpoint"
+                            />
+                        }
+                        fieldId="cloudSource.ocm.endpoint"
+                        touched={touched}
+                        errors={errors}
+                    >
+                        <TextInput
+                            isRequired
+                            type="text"
+                            id="cloudSource.ocm.endpoint"
+                            name="cloudSource.ocm.endpoint"
+                            value={values.cloudSource.ocm?.endpoint}
+                            onChange={(event, value) => onChange(value, event)}
+                            onBlur={handleBlur}
+                            isDisabled={!isEditable}
+                        />
+                    </FormLabelGroup>
+                    {!isCreating && isEditable && (
+                        <FormLabelGroup
+                            fieldId="updateCredentials"
+                            helperText="Enable this option to replace currently stored credentials (if any)"
+                            errors={errors}
+                        >
+                            <Checkbox
+                                label="Update stored credentials"
+                                id="updateCredentials"
+                                isChecked={values.updateCredentials}
+                                onChange={(event, value) => onUpdateCredentialsChange(value, event)}
+                                onBlur={handleBlur}
+                                isDisabled={!isEditable}
+                            />
+                        </FormLabelGroup>
+                    )}
+                    <FormLabelGroup
+                        isRequired={values.updateCredentials}
+                        label="Client ID"
```

Comment:
Here is an alternative that follows current pattern and lint rules:
![Client_ID](https://github.com/user-attachments/assets/fb94961b-a9f6-4cd2-924a-39318f13641c)


```tsx
                        label="Client ID"
                        labelIcon={
                            <IntegrationHelpIcon
                                hasAutoWidth
                                helpTitle="Service account client ID"
                                helpText={
                                    <Flex direction={{ default: 'column' }}>
                                        <FlexItem>
                                            Client identifier for a{' '}
                                            <ExternalLink>
                                                <a
                                                    href="https://console.redhat.com/iam/service-accounts"
                                                    target="_blank"
                                                    rel="noopener noreferrer"
                                                >
                                                    Red Hat service account
                                                </a>
                                            </ExternalLink>
                                        </FlexItem>
                                        <FlexItem>
                                            The service account must belong to a group with the{' '}
                                            <strong>Subscription Viewer</strong> role.
                                        </FlexItem>
                                        <FlexItem>
                                            For more information, see{' '}
                                            <ExternalLink>
                                                <a
                                                    href="https://docs.redhat.com/en/documentation/openshift_cluster_manager/1-latest/html-single/managing_clusters/index#assembly-user-management-ocm"
                                                    target="_blank"
                                                    rel="noopener noreferrer"
                                                >
                                                    Configuring access to clusters in OpenShift
                                                    Cluster Manager
                                                </a>
                                            </ExternalLink>
                                        </FlexItem>
                                        <FlexItem>
                                            <em>
                                                Service accounts are the preferred authentication
                                                method over the deprecated API token.
                                            </em>
                                        </FlexItem>
                                    </Flex>
                                }
                                ariaLabel="Help for client ID"
                            />
                        }
```

In particular, **For more information, see** is convention for OpenShift docs and many other style guides.

Requires the following changes in IntegrationHelpIcon.tsx file:
1. Add `hasAutoWidth?: boolean;` to type.
2. Add `hasAutoWidth,` to function arguments.
3. Add `hasAutoWidth={hasAutoWidth}` as third prop of `Popover` element.

Reply:
done, see [2754468](https://github.com/stackrox/stackrox/pull/12277/commits/27544685b601615d3fd113fbc08622e40c1599e3)